tests/dummy/app/pods/details/controller.js

```diff
@@ -0,0 +1,5 @@
+import Ember from 'ember'
+
+export default Ember.Controller.extend({
+  defaultRoute: 'details.views.maps'
```

Comment:
I think we need more generic tabs/svgs for this  - since it's a user, maybe "Profile" and "Preferences" for the views and "Devices", "Subscriptions" and "Friends" for related?